src/test/java/org/prgrms/cream/domain/user/controller/UserControllerTest.java

```diff
@@ -0,0 +1,397 @@
+		String size = "265";
+		String address = "Seoul";
+
+		userSignUpRequest = new UserSignUpTestRequest(
+			nickname,
+			email,
+			phone,
+			size,
+			address
+		);
+
+		// when
+		ResultActions result = mockMvc.perform(post("/users")
+												   .contentType(MediaType.APPLICATION_JSON)
+												   .content(objectMapper.writeValueAsString(
+													   userSignUpRequest)
+												   )
+		);
+		// then
+		result
+			.andExpect(
+				status().isOk()
+			)
+			.andDo(
+				print()
+			);
+	}
+
+	@DisplayName("회원생성_중복예외_테스트")
+	@Test
+	void saveUserMethodDuplicateExceptionTest() throws Exception {
+		String nickname = "nickName1";
+		String email = "email2@email.com";
+		String phone = "01012341234";
+		String size = "265";
+		String address = "Seoul";
+
+		userSignUpRequest = new UserSignUpTestRequest(
+			nickname,
+			email,
+			phone,
+			size,
+			address
+		);
+
+		ResultActions result = mockMvc.perform(post("/users")
+												   .contentType(MediaType.APPLICATION_JSON)
+												   .content(
+													   objectMapper.writeValueAsString(
+														   userSignUpRequest)
+												   )
+		);
+
+		// then
+		result
+			.andExpect(
+				status().is4xxClientError()
+			)
+			.andExpect(res -> assertTrue(res
+											 .getResolvedException()
+											 .getClass()
+											 .isAssignableFrom(DuplicateUserException.class)
+					   )
+			)
+			.andDo(
+				print()
+			);
+	}
+
+	@DisplayName("회원생성_예외_테스트")
+	@Test
+	void saveUserMethodValidExceptionTest() throws Exception {
+		String nickname = "nickName1";
+		String email = "email1";
+		String phone = "01012341234";
+		String size = "265";
+		String address = "Seoul";
+
+		userSignUpRequest = new UserSignUpTestRequest(
+			nickname,
+			email,
+			phone,
+			size,
+			address
+		);
+
+		ResultActions result = mockMvc.perform(post("/users")
+												   .contentType(MediaType.APPLICATION_JSON)
+												   .content(objectMapper.writeValueAsString(
+													   userSignUpRequest)
+												   )
+		);
+
+		// then
+		result
+			.andExpect(
+				status().is5xxServerError()
+			)
+			.andExpect(
+				res -> assertTrue(
+					res
+						.getResolvedException()
+						.getClass()
+						.isAssignableFrom(ConstraintViolationException.class)
+				)
+			)
+			.andDo(
+				print()
+			);
+	}
+
+	@DisplayName("회원정보수정_테스트")
+	@Test
+	void updateUserTest() throws Exception {
+		Long userId = 2L;
+		// given
+		String option = "nickname";
+		String value = "updatedNickname";
+
+		userUpdateTestRequest = new UserUpdateTestRequest(
+			option,
+			value
+		);
+
+		// when
+		ResultActions result = mockMvc.perform(patch("/users/{id}", userId)
+												   .contentType(MediaType.APPLICATION_JSON)
+												   .content(objectMapper.writeValueAsString(
+													   userUpdateTestRequest)
+												   )
+		);
+
+		// then
+		result
+			.andExpect(
+				status().isOk()
+			)
+			.andExpect(
+				jsonPath("$.data")
+					.value(2L)
```

Comment:
```suggestion
					.value(userId)
```

어차피 같은 값이 나온다고 예상되면 재사용하는 건 어떨까요?

Reply:
a0338d3 : 수정했습니다!! 이 부분을 놓쳤군요 ㅎㅎ